packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -1774,6 +1774,7 @@ export class Task {
             this.hasTaskProblemMatchers = false;
         }
         this.isTaskBackground = false;
+        this.presentationOptions = Object.create(null);
```

Comment:
If `presentationOptions` is always set in the constructor, should the typings for the field not include `undefined`?

Reply:
you are right, 'undefined' should not be an alternative